plugins/prefect/access_key_test.go

```diff
@@ -0,0 +1,41 @@
+package prefect
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAccessKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, AccessKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.APIKey: "pnu_dOXnQZOBq4Sst6hIesrdcbYVbnu1XEXAMPLE",
+			},
+			ExpectedOutput: sdk.ProvisionOutput{
+				Environment: map[string]string{
+					"PREFECT_API_KEY": "pnu_dOXnQZOBq4Sst6hIesrdcbYVbnu1XEXAMPLE",
+				},
+			},
```

Comment:
Let's include the URL in this as well:

```suggestion
			ItemFields: map[sdk.FieldName]string{
				fieldname.APIKey: "pnu_dOXnQZOBq4Sst6hIesrdcbYVbnu1XEXAMPLE",
				fieldname.URL:      "https://api.prefect.cloud/api/accounts/12334567890/workspaces/1234567890",
			},
			ExpectedOutput: sdk.ProvisionOutput{
				Environment: map[string]string{
					"PREFECT_API_KEY": "pnu_dOXnQZOBq4Sst6hIesrdcbYVbnu1XEXAMPLE",
					"PREFECT_API_URL": "https://api.prefect.cloud/api/accounts/12334567890/workspaces/1234567890",
				},
			},
```